pkg/connect/state/state.go

```diff
@@ -375,3 +409,21 @@ func (g *WorkerGroup) Sync(ctx context.Context, groupManager WorkerGroupManager,
 
 	return nil
 }
+
+type WorkerCapacity struct {
+	Total         int64
+	Available     int64
+	CurrentLeases []string
```

Comment:
Do we currently access this? I'm wondering because this could get really large for some users 🤞 We may want to omit this and just return the counts.